soroban-env-host/src/builtin_contracts/stellar_asset_contract/balance.rs

```diff
@@ -248,7 +248,7 @@ pub(crate) fn write_authorization(
     }
 }
 
-// TODO: Metering analysis
+// TODO: Metering: covered by components.
```

Comment:
Nit: The TODO should be removed. This applies to a few places in the file.

Reply:
Pushed a change fixing this